src/pages/iou/SplitBillDetailsPage.tsx

```diff
@@ -62,6 +62,7 @@ function SplitBillDetailsPage({route, report, reportAction}: SplitBillDetailsPag
     const payeePersonalDetails = personalDetails?.[actorAccountID];
     const participantsExcludingPayee = participants.filter((participant) => participant.accountID !== reportAction?.actorAccountID);
 
+    const isDistanceRequestTransaction = isDistanceRequest(transaction);
```

Comment:
```suggestion
    const isDistanceRequest = isDistanceRequest(transaction);
```

NIT: The transaction word looks redundant to me.